README.md

```diff
@@ -37,39 +37,33 @@ Here are a few key pages you might be interested in:
   - [Transfers](https://docs.tigerbeetle.com/reference/transfers)
   - [Operations](https://docs.tigerbeetle.com/reference/operations)
 
-## QuickStart
+## Quickstart
 
 TigerBeetle is easy to run with or without Docker, depending on your
-preference.
+preference. First, we'll cover running the [Single
+Binary](#single-binary). And below that is how to run [with
+Docker](#with-docker).
 
-### With Docker
+### Single Binary
 
-First provision TigerBeetle's data directory.
+Install TigerBeetle by grabbing the latest release from
+GitHub. For example, on Linux run:
 
 ```bash
-$ docker run -v $(pwd)/data:/data ghcr.io/tigerbeetledb/tigerbeetle \
-    format --cluster=0 --replica=0 /data/0_0.tigerbeetle
-info(io): creating "0_0.tigerbeetle"...
-info(io): allocating 660.140625MiB...
+$ curl -LO https://github.com/tigerbeetledb/tigerbeetle/releases/download/2022-11-16-weekly/tigerbeetle-Linux-x64-2022-11-16-weekly.zip
+$ unzip tigerbeetle-Linux-x64-2022-11-16-weekly.zip 
```

Comment:
https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases says there is a way to link to "latest" but I can't find it so maybe the docs are out of date. :confused:

Reply:
I have never been able to figure this out either without multiple API calls.

Reply:
K, I guess is their docs are just out of date. :shrug: